src/main/target/ANYFCF7/target.h

```diff
@@ -24,6 +24,7 @@
 #define USBD_PRODUCT_STRING "AnyFCF7"
 
 #define USE_DSHOT
+#define USE_ESC_TELEMETRY
```

Comment:
Should we just go
```
#ifdef USE_DSHOT
#define USE_ESC_TELEMETRY
#endif
```

After all all targets that can run DShot and have a serial port can run ESC telemetry.